### PR TITLE
Announcement blog post - draft for review

### PR DESCRIPTION
**Pull request by isabela-pf**
One of the tasks for announcing these events is to propose a blog post on the [Jupyter blog](https://blog.jupyter.org/). I'd like to get some review here while I figure out who to contact in the Jupyter communications side for further review.

Some notes
- There are empty links for now. Once we get #4 and #2 reviewed and merged, that should fill all the empty links.
- There's no image here, though I am planning to make one for the post.
- I'm not certain what the useful information in an announcement is, so feedback on missing content is absolutely welcome.

I don't think this has to be merged. I plan to delete this PR and branch after approval unless someone thinks its useful to have the blog content live on this repo. (Maybe for archive purposes? I don't know.) 

Thanks in advance for the review! 🌻

### CODE REVIEW

File: blog-draft.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Join us for the Jupyter Accessibility Workshops (Part 1)
+
+At long last, Jupyter accessibility workshops are here! This is a series of [Jupyter community workshops](https://blog.jupyter.org/jupyter-community-workshops-cbd34ac82549) aimed at increasing accessibility knowledge within the ecosystem and making Jupyter projects more usable for disabled people.
```
Comment (trallard) [inline blog-draft.md:3]: > accessibility knowledge

Thoughts on replacing by accessibility literacy/awareness?

File: blog-draft.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+The goal is to make accessibility efforts more prominent across the many Jupyter projects. Much of what we discuss will relate to the [Web Content Accessibility Guidelines](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) and how to apply them in open source, scientific spaces where this work has historically been neglected.
```
Comment (trallard) [inline blog-draft.md:5]:
```suggestion
The goal is to make existing accessibility efforts more prominent across the many Jupyter projects. Much of what we discuss will relate to the [Web Content Accessibility Guidelines](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) and how to apply them in open source, scientific spaces where this work has historically been neglected.
```

File: blog-draft.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+These workshops are a series of events proposed for and supported by the [third community workshop cycle](https://blog.jupyter.org/jupyter-community-workshops-call-for-proposals-for-jan-aug-2020-710f687e30f4).
+
+## What are these events?
+
+These workshops are a series of four events alternating between learning from experts and contributing to projects.
+
+They've been designed by listening to community requests and combining event types to include people of different skills and across communities. In total, they combine Jupyter experts and accessibility experts, learning and action, and working with old friends and making new connections.
+
+### Part 1:
```
Comment (trallard) [inline blog-draft.md:15]: I'd suggest add the date here for those that skim through texts

**Review by trallard — APPROVED** [review]

**Review by trallard — APPROVED** [review]

**Comment by trallard** [comment]
Are we still waiting for Chris's review on this? 

 Did not want to tag him unless necessary 👀

**Comment by isabela-pf** [comment]
I pointed him to this PR via [jupyter/accessibility #43](https://github.com/jupyter/accessibility/issues/43) because I don't know a better way to get review from someone who has power in the Jupyter blog. 

I do know we'll need review from someone. I can also ask in the JupyterLab meeting today since I will be going there anyways.

**Comment by isabela-pf** [comment]
Okay, I got more info on the process in the JupyterLab meeting. This needs to be on Medium for review, so I'll close this PR and move the draft there so we can move forward. Thanks again for the review!

**Pull request closed without merging**